co-log/src/Colog/Message.hs

```diff
@@ -127,27 +131,31 @@ showSourceLoc cs = square showCallStack
 types. The type family is open so you can add new instances.
 -}
 type family FieldType (fieldName :: Symbol) :: Type
-type instance FieldType "thread-id" = ThreadId
-type instance FieldType "utc-time"  = UTCTime
+type instance FieldType "threadId" = ThreadId
+type instance FieldType "utcTime"  = UTCTime
```

Comment:
Maybe I should name strings in the same way as types to make implementation better compatible with the ecosystem? 🤔

Reply:
I agree with the same name as the type 👍

Reply:
I just realised that it's not possible to give the same name as type because labels can't start with capital letter 🙂 So let's keep lowerCamelCase